tests/utils.test.js

```diff
@@ -0,0 +1,28 @@
+const {convertToJson} = require("../scripts/utils")
+
+describe('convertToJson', () => {
+  test('should return JSON object if input is valid JSON string', () => {
+    const jsonString = '{"name": "AsyncAPI", "age": 5}';
```

Comment:
Can you create an actual json file (for tests only), and read its content on the go rather than hard-coding JSON strings like this?

If `fs` module isn't working in jest, you can even create js objects and use `json.stringify()` to convert objects to JSON String.

Reply:
you can create `fixtures` to store the static data that are needed in tests.

Reply:
done